File: example-base-28/nodejs-utilities-playground-v2/src/dsa/sorting/selection-sort.js
function selectionSort(numbers) {
  const result = [...numbers];
  for (let i = 0; i < result.length - 1; i++) {
    let minIndex = i;
    for (let j = i + 1; j < result.length; j++)
      if (result[j] < result[minIndex]) minIndex = j;
    [result[i], result[minIndex]] = [result[minIndex], result[i]];
  }
  return result;
}
module.exports = { selectionSort };

File: example-base-29/python-algorithms-playground/algorithms/sorting/selection_sort.py
def selection_sort(arr):
    a = arr.copy()
    for i in range(len(a)):
        min_index = i
        for j in range(i + 1, len(a)):
            if a[j] < a[min_index]:
                min_index = j
        a[i], a[min_index] = a[min_index], a[i]
    return a

if __name__ == "__main__":
    print(selection_sort([64, 25, 12, 22, 11]))
